presto-kafka/src/test/java/io/prestosql/plugin/kafka/util/TestingKafka.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka.util;
+
+import com.google.common.collect.ImmutableMap;
+import kafka.producer.ProducerConfig;
+import org.testcontainers.containers.KafkaContainer;
+
+import java.io.Closeable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import static io.prestosql.plugin.kafka.util.TestUtils.toProperties;
+import static org.testcontainers.containers.KafkaContainer.KAFKA_PORT;
+
+public class TestingKafka
+        implements Closeable
+{
+    private final KafkaContainer kafkaContainer;
+
+    public TestingKafka()
+    {
+        kafkaContainer = new KafkaContainer("5.2.1");
```

Comment:
am i correct we were using 1.1.1 for testing before?
please upgrade version and refactor tests separately
or, if not possible, reflect this in cmt msg

like in https://github.com/prestosql/presto/pull/2163#discussion_r352634164

Reply:
This is a bit more complicated. See https://docs.confluent.io/current/installation/versions-interoperability.html

Docker version is a version of confluent. So it looks like we were using 4.1.x kind of confluent version.